namer/namer.go

```diff
@@ -30,13 +46,61 @@ type Namer interface {
 	ParseNames(names []string) []Key    // Convert names into keys.
 }
 
-// Generator generates signer K/V pairs.
-// Implementation should use `generic` and will used for strong typing of the solution.
-type Generator[T any] interface {
-	Generate(name string, value T) ([]kv.KeyValue, error)
+// DefaultNamer represents default namer.
+type DefaultNamer struct {
+	prefix string
+}
+
+// NewDefaultNamer returns new DefaultNamer object.
+func NewDefaultNamer(prefix string) *DefaultNamer {
+	return &DefaultNamer{
+		prefix: prefix,
+	}
+}
+
+// GenerateNames generates set of names from basic name.
+func (n *DefaultNamer) GenerateNames(name string) []string {
+	return []string{
+		n.prefix + "/" + name,
+		n.prefix + "/" + hashName + "/" + name,
+		n.prefix + "/" + sigName + "/" + name,
+	}
 }
 
-// Validator validates and build the object from K/V.
-type Validator[T any] interface {
-	Validate(pairs []kv.KeyValue) (T, error)
+// ParseNames returns set of Keys with different types.
+func (n *DefaultNamer) ParseNames(names []string) []Key {
+	keys := make([]Key, 0, namesNumber)
+
+	for _, name := range names {
+		var key Key
+
+		// Remove prefix.
+		result := strings.ReplaceAll(name, n.prefix, "")
```

Comment:
```suggestion
		result, _ := strings.CutPrefix(name, n.prefix)
```
The original will remove not only prefix, but all occurrences of `n.prefix` substring.